components/raftstore/src/store/worker/split_check.rs

```diff
@@ -219,32 +220,40 @@ impl Display for Task {
     }
 }
 
-pub struct Runner<E, S>
-where
-    E: KvEngine,
-{
-    engine: E,
+pub struct Runner<EK: KvEngine, S> {
+    // We can't just use `TabletRegistry` here, otherwise v1 may create many
+    // invalid records and cause other problems.
+    engine: Either<EK, TabletRegistry<EK>>,
```

Comment:
why term `Tablet` is introduced in raftstore v1?

Reply:
V2 reuses the worker to do automatically split.